integration_tests/mockito-experimental/src/test/java/org/robolectric/integrationtests/mockito/experimental/MockitoSpyTest.java

```diff
@@ -22,4 +27,14 @@ public void spyActivity_hasSameBaseContext() {
     Activity spyActivity = spy(activity);
     assertThat(activity.getBaseContext()).isEqualTo(spyActivity.getBaseContext());
   }
+
+  @Test
+  public void spyContext_canSpyGetSystemService() {
+    Context context = spy(RuntimeEnvironment.getApplication());
+    InputMethodManager expected = mock(InputMethodManager.class);
+    when(context.getSystemService(Context.INPUT_METHOD_SERVICE)).thenReturn(expected);
+    InputMethodManager actual =
+        (InputMethodManager) context.getSystemService(Context.INPUT_METHOD_SERVICE);
+    assertThat(actual).isSameInstanceAs(expected);
```

Comment:
Assuming this assertion is run, does it fail or does it pass? If it fails, we should add `@Ignore` for now.

I am curious if this passed in previous versions of Robolectric. If it did, we could try a git bisect.

Reply:
I think it should works. I found the real problem of Flutter is caused by Android 12's changes for `Presentation`. I will retest it after we enable related test modules.

Reply:
I have tested it locally, and it works.